src/plugins/unified_search/public/query_string_input/query_bar_top_row.tsx

```diff
@@ -126,6 +126,20 @@ const SharingMetaFields = React.memo(function SharingMetaFields({
 export const QueryBarTopRow = React.memo(
   function QueryBarTopRow(props: QueryBarTopRowProps) {
     const isMobile = useIsWithinBreakpoints(['xs', 's']);
+    const [isXXLarge, setIsXXLarge] = useState<boolean>(false);
+
+    useEffect(() => {
+      function handleResize() {
+        setIsXXLarge(window.innerWidth >= 1440);
```

Comment:
@stratoula , none of the EUI breakpoint hooks/functions allow for custom sizing and we don't go past 1200px. But this is the lowest I would go to show the full button. I hope it's ok to reference `window` here.

Reply:
This looks fine :)

Reply:
Awesome!